docs/README.md

```diff
@@ -798,6 +798,32 @@ Que.job_middleware.push(
 )
 ```
 
+#### Existing Middleware
+
+Que ships with middleware to expose job metrics using ActiveSupport notifications to subscribe to it you can implelent the following
+
+```ruby
+::ActiveSupport::Notifications.subscribe("que_job.worked") do |message, started, finished, labels|
```

Comment:
I just [looked this up](https://api.rubyonrails.org/classes/ActiveSupport/Notifications.html#module-ActiveSupport::Notifications-label-Subscribers), and it seems that `.subscribe` doesn't have monotonic timestamps; `.monotonic_subscribe` does. Whichever you want to use here is fine, but let's document it correctly.